### ISSUE TITLE
Add support for proposed Element.closest

### ISSUE THREAD

**Issue by tmorehouse** [inexorabletash/polyfill#127]
The Element.closest() method provides a way to search up the dom tree for a node  matching a CSS selector

from https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

```
if (window.Element && !Element.prototype.closest) {
    Element.prototype.closest = 
    function(s) {
        var matches = (this.document || this.ownerDocument).querySelectorAll(s),
            i,
            el = this;
        do {
            i = matches.length;
            while (--i >= 0 && matches.item(i) !== el) {};
        } while ((i < 0) && (el = el.parentElement)); 
        return el;
    };
}
```

**Comment by alexandernanberg** [comment]
I would like to see this added as well!

**Comment by alexandernanberg** [comment]
@inexorabletash Can you publish a new version now that my PR is merged? :)

**Comment by inexorabletash** [comment]
Done, thanks for the prompt!

**Comment by inexorabletash** [comment]
Resolved by b46b5c7e1d4d198a5b450da686a372795c0d5238
